runelite-client/src/main/java/net/runelite/client/plugins/attackstyles/AttackStylesConfig.java

```diff
@@ -98,24 +98,24 @@ default boolean warnForMagic()
 	}
 
 	@ConfigItem(
-		keyName = "hideAutoRetaliate",
-		name = "Hide auto retaliate",
-		description = "Hide auto retaliate from the combat options tab",
+		keyName = "warnForAutoRetaliate",
+		name = "Warn for auto retaliate",
+		description = "Show warning when auto retaliate is selected",
 		position = 7
 	)
-	default boolean hideAutoRetaliate()
+	default boolean warnForAutoRetaliate()
 	{
 		return false;
```

Comment:
why is auto retaliates format specifically warn?

Reply:
See thread above. Changed so that auto retaliate is handled same as other buttons. There's 1 new setting in which you can specify if you want to: do nothing, be warned, hide buttons or lock buttons. 

Now that you mention it: I might need to add a chatmessage when the option is set to 'warn' and auto retaliate switches.